### DISCUSSION TITLE
关于pantherX2把系统安装到emmc的问题

### DISCUSSION THREAD

**Discussion by darkelfzdy** [ophub/amlogic-s9xxx-armbian#2104] [Rockchip]
请教各位大佬，今天pantherX2（rk3566）到货，我已经通过tf卡安装Armbian成功启动运行，现在正在琢磨把Armbian输入emmc，有几个问题请教一个各位：
我仔细看了一下教程，虽然单独列出了Rockchip 系列，但是只说了Radxa-Rock5B、电犀牛 R66S、电犀牛 R68S、贝壳云、我家云的说明，但没有说明其他的rk是参照上面某一个还是就使用哪个armbian-install命令安装，我就想确认一下，pantherX2从tf卡刷到emmc是就用armbian-install，还是要用线刷？

**Comment by darkelfzdy** [comment]
突然发现pantherX2安装的armbian根本就没有armbian-install命令，看来只能线刷

**Comment by ophub** [comment]
把armbian.img文件上传到tf卡的某个目录，如/root下，然后dd写入emmc，可以看rockchip的tf写入emmc的做法。
